3rd-party/woocommerce.php

```diff
@@ -23,14 +31,14 @@ function jetpack_woocommerce_integration() {
 	}
 }
 
-/*
+/**
  * Make sure the social sharing icons show up under the product's short description
  */
 function jetpack_woocommerce_social_share_icons() {
 	if ( function_exists( 'sharing_display' ) ) {
 		remove_filter( 'the_content', 'sharing_display', 19 );
 		remove_filter( 'the_excerpt', 'sharing_display', 19 );
-		echo sharing_display();
+		sharing_display( '' . true );
```

Comment:
This doesn't seem right. Did you mean
```suggestion
		sharing_display( '', true );
```